src/filters/valve.cpp

```diff
@@ -83,19 +85,39 @@ Try<string> getFormValue(
 }
 
 class ValveFilterEndpointProcess
-  : public Process<ValveFilterEndpointProcess> {
+  : public ProtobufProcess<ValveFilterEndpointProcess> {
  public:
   explicit ValveFilterEndpointProcess(const Tag& _tag, bool _opened)
     : tag(_tag),
       ProcessBase(getValveProcessBaseName(_tag.TYPE())),
       opened(_opened),
-      limiter(2, Seconds(1)) {}  // 2 permits per second.
+      // 2 permits per second.
+      limiter(2, Seconds(1)) {
+    switch (this->tag.TYPE()) {
```

Comment:
Maybe we should resolve it by having multiple comm buses?
Or topic-based subscription.